system/Cookie/CookieInterface.php

```diff
@@ -106,7 +106,7 @@ public function getValue(): string;
 	 *
 	 * @return integer
 	 */
-	public function getExpiresTimestamp(): int;
+	public function getExpires(): int;
```

Comment:
I'll accept your change for `withExpires($expires)`. But for here, can we reach a compromise and meet halfway?

I want this to be at least `getExpiresTime()` to differentiate it clearly from `getExpiresString()`.

Reply:
done